lib/tasks/package_types.rake

```diff
@@ -27,4 +27,27 @@ namespace :goodcity do
     end
   end
 
+  #rake goodcity:update_allow_stock_to_stockit_codes_status
+  desc 'Update PackageType `allow_stock` according to `Codes.status` from Stockit'
+  task update_allow_stock_to_stockit_codes_status: :environment do
+    STATUS_AND_ALLOW_STOCK_MAPPING = {
+      "Active" => true,
+      "Inactive" => false
+    }
+    codes_json = Stockit::CodeSync.index
+    stockit_codes = JSON.parse(codes_json["codes"]) || {}
+    stockit_codes.each do |code|
+      if active?(code["status"])
```

Comment:
no need to have a method call for this.. 
```suggestion
      if STATUS_AND_ALLOW_STOCK_MAPPING[code["status"]]
```

Reply:
`active?(...)` is much more readable and understandable for me. I think having a mapping for what is essentially a boolean value is overkill

Reply:
I added `active?` method to make its intention more obvious. I think its fine to with this method.